src/main/java/Utilities/Storage.java

```diff
@@ -0,0 +1,135 @@
+package utilities;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import tasks.Deadlines;
+import tasks.Event;
+import tasks.Task;
+import tasks.ToDos;
+
+public class Storage {
+    private File file;
+
+    public Storage(String path) {
+        this.file = new File(path);
+    }
+
+    /**
+     * Reads from file and returns the resulting TaskList.
+     */
+    public ArrayList<Task> loadTaskListFromFile() {
+        ArrayList<Task> tasks = new ArrayList<>();
+        try {
+            Scanner fileScanner = new Scanner(this.file);
+            while (fileScanner.hasNextLine()) {
+                String line = fileScanner.nextLine();
+                // use "," as delimiters
+                String[] splits = line.split(",");
+                switch (splits[0]) {
+                case "T":
+                    tasks.add(new ToDos(splits[2]));
+                    break;
+                case "D":
+                    tasks.add(new Deadlines(splits[2], splits[3].strip()));
+                    break;
+                case "E":
+                    tasks.add(new Event(splits[2], splits[3].strip(), splits[4].strip()));
+                    break;
+                default:
+                    break;
```

Comment:
I like how you include a default case here for clarity and code safety purpose! 😄